maas-schemas-ts/src/core/modes/MODE_SHARED_E_BICYCLE.ts

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
How did that get here? It was generated when I ran the convert command but why it wasn't added earlier???